### PR TITLE
Update script to set author-id and owner-id

### PR DESCRIPTION
**Pull request by annando**
The author-id and owner-id will now be set to good values in an update script. So feed items will appear again.

### CODE REVIEW

File: src/Database/DBStructure.php [hunk 1]
```diff
@@ -207,7 +207,7 @@ private static function printUpdateError($message) {
 	public static function update($verbose, $action, $install = false, array $tables = null, array $definition = null) {
 		if ($action && !$install) {
 			Config::set('system', 'maintenance', 1);
-			Config::set('system', 'maintenance_reason', L10n::t(': Database update', DBM::date().' '.date('e')));
+			Config::set('system', 'maintenance_reason', L10n::t('%1: Database update', DBM::date().' '.date('e')));
```
Comment (MrPetovan) [inline src/Database/DBStructure.php]: Didn't you mean `%s`?

File: update.php [hunk 2]
```diff
@@ -184,3 +186,43 @@ function update_1247() {
 	// Make sure we install the new renamed ones
 	Addon::reload();
 }
+
+function update_1260() {
+	Config::set('system', 'maintenance', 1);
+	Config::set('system', 'maintenance_reason', L10n::t('%1: Updating author-id and owner-id in item and thread table. ', DBM::date().' '.date('e')));
```
Comment (MrPetovan) [inline update.php]: Same question, I've never seen `%1` in `sprintf`.

File: src/Database/DBStructure.php (hunk 1 above)
Comment (annando) [reply src/Database/DBStructure.php]: Oops ... Yeah. The placeholders in the language at our company are looking that way.

**Comment by MrPetovan** [comment]
Related to #4994

**Comment by annando** [comment]
Yeah, I'm done with it.

**Comment by AlfredSK** [comment]
After pulling this I got that...

````
PHP Fatal error:  Uncaught Error: Class 'L10n' not found in /var/www/html/libranet/update.php:192                                                                        Stack trace:
#0 /var/www/html/libranet/boot.php(790): update_1260()                                            #1 /var/www/html/libranet/boot.php(760): run_update_function(1260)                                #2 /var/www/html/libranet/src/Worker/DBUpdate.php(18): update_db(Object(Friendica\App))
#3 [internal function]: Friendica\Worker\DBUpdate::execute()
#4 /var/www/html/libranet/src/Core/Worker.php(351): call_user_func_array('Friendica\\Worke...', Array)                                                                                              #5 /var/www/html/libranet/src/Core/Worker.php(243): Friendica\Core\Worker::execFunction(Array, 'DBUpdate', Array, true)                                                                             #6 /var/www/html/libranet/src/Core/Worker.php(105): Friendica\Core\Worker::execute(Array)
#7 /var/www/html/libranet/bin/worker.php(59): Friendica\Core\Worker::processQueue(false)
#8 {main}                                                                                           thrown in /var/www/html/libranet/update.php on line 192
````

**Comment by AlfredSK** [comment]
I guess `use Friendica\Core\L10n;` is missing in update.php.

**Pull request merged**